features/product/managing_products/adding_product.feature

```diff
@@ -0,0 +1,33 @@
+@managing_products
+Feature: Adding a new product
+    In order to extend my merchandise
+    As an Administrator
+    I want to add a new product to the shop
+
+    Background:
+        Given the store is available in "English (United States)"
+        And the store has a base currency "Euro"
+        And I am logged in as an administrator
+
+    @ui
+    Scenario: Adding a new simple product
+        Given I want to create a new simple product
+        When I specify its code as "BOARD_DICE_BREWING"
+        And I name it "Dice Brewing" in "English (United States)"
+        And I make it priced at "€10.00"
```

Comment:
I set its price to "€10.00"

Wdyt? Simpliest and correct. When you work on a product, you set its price, that's it I guess. :)